src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java

```diff
@@ -0,0 +1,172 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.backpage.dispatch.DispatchData;
+import com.github.manolo8.darkbot.backpage.dispatch.InProgress;
+import com.github.manolo8.darkbot.backpage.dispatch.Retriever;
+import com.github.manolo8.darkbot.backpage.entities.galaxy.GalaxyInfo;
+import com.github.manolo8.darkbot.utils.IOUtils;
+import com.github.manolo8.darkbot.utils.http.Method;
+import org.intellij.lang.annotations.Language;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.BiConsumer;
+import com.github.manolo8.darkbot.backpage.dispatch.BiIntConsumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DispatchManager {
+    private final Main main;
+    private final DispatchData data;
+    private long lastDispatcherUpdate;
+
+    DispatchManager(Main main) {
+        this.main = main;
+        this.data = new DispatchData();
+    }
+
+    public DispatchData getData() {
+        return data;
+    }
+
+    public boolean update(int expiryTime) {
+        try {
+            return update(main.backpage, expiryTime);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+    public boolean update(BackpageManager manager, int expiryTime) throws Exception {
+        if (System.currentTimeMillis() <= lastDispatcherUpdate + expiryTime) return false;
+        String page = manager.getConnection("indexInternal.es?action=internalDispatch", Method.GET, 10_000).getContent();
```

Comment:
The 10_000 here is pretty arbitrary, it means there can never be 2 reloads of the page in less than 10 seconds which i don't think makes much sense, in-game you can definetly reload faster than that.
Lower that to 1s if you still want to keep some "rate-limit"